app/src/main/java/io/intrepid/contest/rest/MockRestApi.java

```diff
@@ -90,12 +91,21 @@ private RedeemInvitationResponse getValidRedeemInvitationResponse(ParticipationT
         // Participant changed
         numGetContestStatusCallsForParticipant = 0;
 
-        if (code.equals(TEST_JUDGE_CODE)) {
-            return Observable.just(getValidRedeemInvitationResponse(ParticipationType.JUDGE));
+        switch (code) {
```

Comment:
In a previous iteration, I actually made some other dummy codes like LATE_JUDGE to simulate a user entering a judge code for a contest that already ended so I had some other cases here, but after we spoke, I realized that is out of scope for this. I will revert this back to a regular if statement.